frame/curve-amm/src/tests.rs

```diff
@@ -152,6 +171,26 @@ fn get_y_j_greater_than_n() {
 	assert_eq!(result, None);
 }
 
+#[test]
+fn test_get_price() {
+	new_test_ext().execute_with(|| {
+		let assets = vec![MockCurrencyId::USDC, MockCurrencyId::USDT];
```

Comment:
can remove Mock form Currency, than easy to work with code in real runtme and in tests

Reply:
I will do it in separate PR.